BitwardenShared/UI/Platform/Application/Views/BitwardenTextField.swift

```diff
@@ -137,28 +137,29 @@ struct BitwardenTextField<FooterContent: View, TrailingContent: View>: View {
     }
 
     /// The text field.
+    /// After some investigation, we found that .id(..) needs to be the final modifier
+    /// to avoid breaking accessibilityIds used on our mobile automation test suite
```

Comment:
⛏️ I'd move this just before the first TextField using the workaround given that it's not a comment for the whole property.
Additionally, from what I see `.id(...)` doesn't have to be the final modifier; it just needs to be after `.accessibilityIdentifier(...)`, right?